### ISSUE TITLE
Lack of documentation resulting `TemplateSyntaxError: Encountered unknown tag 'assets'`

### ISSUE THREAD

**Issue by kmonsoor** [kdeldycke/plumage#169]
Platform
========
Ubuntu 20.04

Steps to reproduce
==================
 * init a barebone Pelican project using `pelican quickstart`. Runs successfully.
 * Install  via Pip & update the `pelicanconf.py` with installation step from this repo

Issue 1
-------
>poetry run pelican --verbose ./content

Where is Poetry dependency coming from? Neither Pelican nor Plumage installation steps is mentioning it.

Issue 2
-------
After installing Poetry, it's required `pyproject.toml` (copying from [kdeldycke blog](https://github.com/kdeldycke/kevin-deldycke-blog/blob/main/pyproject.toml) then successfully running it, below is the output of running `poetry show --tree` (only including the `Plumage` & webassets part.)

```
plumage 2.4.0 Clean and tidy theme for Pelican.
├── closure ^20191111
├── cssmin ^0.2.0
├── libsass ^0.20.1
│   └── six *
├── markdown ^3.3.3
├── pelican ^4.2.0
│   ├── blinker >=1.4
│   ├── docutils >=0.16
│   ├── feedgenerator >=1.9
│   │   └── pytz >=0a
│   ├── jinja2 >=2.7
│   │   └── markupsafe >=2.0
│   ├── pygments >=2.6
│   ├── python-dateutil >=2.8
│   │   └── six >=1.5
│   ├── pytz >=2020.1 (circular dependency aborted here)
│   ├── rich >=10.1
│   │   ├── colorama >=0.4.0,<0.5.0
│   │   ├── commonmark >=0.9.0,<0.10.0
│   │   └── pygments >=2.6.0,<3.0.0 (circular dependency aborted here)
│   └── unidecode >=1.1
├── pelican-webassets ^1.0.0
│   ├── pelican *
│   │   ├── blinker >=1.4
│   │   ├── docutils >=0.16
│   │   ├── feedgenerator >=1.9
│   │   │   └── pytz >=0a
│   │   ├── jinja2 >=2.7
│   │   │   └── markupsafe >=2.0
│   │   ├── pygments >=2.6
│   │   ├── python-dateutil >=2.8
│   │   │   └── six >=1.5
│   │   ├── pytz >=2020.1 (circular dependency aborted here)
│   │   ├── rich >=10.1
│   │   │   ├── colorama >=0.4.0,<0.5.0
│   │   │   ├── commonmark >=0.9.0,<0.10.0
│   │   │   └── pygments >=2.6.0,<3.0.0 (circular dependency aborted here)
│   │   └── unidecode >=1.1
│   └── webassets *
```

With this state of Poetry, when I run `poetry run pelican --verbose ./content`, I get output like below. 

```
[20:38:27] ERROR    Cannot load plugin `pelican.plugins.webassets`                                      log.py:96
                    Cannot import plugin `pelican.plugins.webassets`
[20:38:28] WARNING  Watched path does not exist: /home/kmx/pelican-demo/content/images                  log.py:91
           CRITICAL TemplateSyntaxError: Encountered unknown tag 'assets'.`
```

How can I overcome this error?

**Comment by kmonsoor** [comment]
after a long back and forth of pip install & uninstall, I found out this is about mentioning the _wrong_ (latest/`4.7.1`) version of `pelican` as required in the latest "official release" of `plumage` which is `2.4.0`.  

The failure to load the plugin `pelican.plugins.webassets` by the latest `pelican` (due to some change of loading mechanism of the plugins and `webassets` not catching up to that), leads to the `assets` tag to remain unrecognizable.

Once I updated the `pyproject.toml` to install the current, latest rev of the main branch of `plumage` **instead** of the latest (`2.4.0`) release available on the PyPI, it seems to solve this issue for me. Btw, this latest `plumage` code requires the `pelican` module to be `4.2.0` which is capable to load the `webassets` plugin.

Also, needed to make sure that the latest (currently `4.7.1`) version of `pelican` isn't accessible of the `poetry run` environment in any way. For that, need to check with `poetry show --tree` command.

**Comment by kmonsoor** [comment]
According to my current findings, during I was migrating my blog build to use Github Actions, it seems like that my previous comment about wrong version of Pelican to be the root cause probably not correct.
But the bit about switching to the `develop` branch of Plumage still holds.

**Comment by kdeldycke** [comment]
I'm sorry @kmonsoor for all the trouble. I neglected that theme too much for the past year and it shows. Even my blog that is based on Plumage is no longer building nor updating. There's a high chance I'll start cleaning-up, stabilize and release new versions of Plumage in the next few months. As time goes by I'm a little bit bothered my stale blog...

Anyway, thanks again for taking the time to carefully test and document all the current shortcomings. In the mean time I've merged #173 to include your instructions in the readme.
